.github/workflows/test.yml

```diff
@@ -27,4 +27,5 @@ jobs:
       run: tox run -e py,report_ci,lint
     - uses: codecov/codecov-action@v3
       with:
+        token: ${{ secrets.CODECOV_TOKEN }}
         fail_ci_if_error: true
```

Comment:
I added `fail_ci_if_error: true` in case I accidentally misconfigured something.  I don't have it set like that on my other repos, I just checked. Most of the time I set no values at all apparently.
```suggestion
```